### PR TITLE
button accept png

### PR DESCRIPTION
**Pull request by landim32**
make the button accept png format

### CODE REVIEW

**Comment by ddelrio1986** [comment]
Maybe jpeg and tiff should be added as well.

File: src/messenger/webim/styles/emagine/templates/chat.tpl [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
```
Comment (gnovaro) [inline src/messenger/webim/styles/emagine/templates/chat.tpl:1]: Use HTML5 Doctype
<!DOCTYPE HTML>

File: src/messenger/webim/styles/emagine/templates/chat.tpl [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+<html>
+<head>
+<title>${msg:chat.window.title.agent}</title>
+<link rel="shortcut icon" href="${webimroot}/images/favicon.ico" type="image/x-icon">
+<link rel="stylesheet" type="text/css" href="${tplroot}/chat.css">
+<link href='http://fonts.googleapis.com/css?family=Ubuntu:400,700' rel='stylesheet' type='text/css'>
+<script type="text/javascript" language="javascript" src="${webimroot}/js/${jsver}/common.js"></script>
```
Comment (gnovaro) [inline src/messenger/webim/styles/emagine/templates/chat.tpl:8]: Remove deprecated tags type="text/javascript" language="javascript"

File: src/messenger/webim/styles/emagine/templates/error.tpl [hunk 3]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<title>${msg:chat.error_page.title}</title>
+<link rel="shortcut icon" href="${webimroot}/images/favicon.ico" type="image/x-icon"/>
+<link rel="stylesheet" type="text/css" href="${tplroot}/chat.css" />
+<style type="text/css">
```
Comment (gnovaro) [inline src/messenger/webim/styles/emagine/templates/error.tpl:7]: Remove document styles, move this to external .css file

File: src/messenger/webim/styles/emagine/templates/error.tpl [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+	-moz-background-inline-policy: -moz-initial;
+}
+.icrnlb { background-position:-40px -15px; width: 15px; height: 15px; }
+.icrnlt { background-position:-40px 0px; width: 15px; height: 15px; }
+.icrnrb { background-position:-55px -15px; width: 15px; height: 15px; }
+.icrnrt { background-position:-55px 0px; width: 15px; height: 15px; }
+.iback { background-position:-41px -30px; width: 25px; height: 26px; }
+</style>
+</head>
+<body bgcolor="#FFFFFF" style="background-image: url(${tplroot}/images/bg.gif); margin: 0px;" text="#000000" link="#C28400" vlink="#C28400" alink="#C28400">
```
Comment (gnovaro) [inline src/messenger/webim/styles/emagine/templates/error.tpl:21]: Remove bgcolor="" and style="" atributes assign css class to apply this changes

**Pull request closed without merging**